### DISCUSSION TITLE
Provide option to use "patience" diff when reviewing PRs

### DISCUSSION THREAD

**Discussion by dylemma** [community/community#16782] [Pull Requests]
I just received a review request for a PR that looks like this in GitHub: 
![image](https://user-images.githubusercontent.com/1355958/168824301-9b8fe353-be22-4292-98b4-8ade9c439915.png)

The author insisted that he had only *added* code, and when I investigated, I found that generating the diff via `git diff --patience` confirmed his claim; he had added around 1100 lines and made 0 deletions.

It would be very helpful, in situations like this, to be able to toggle the PR changes view to "patience" mode so I can avoid going back to the command line, and be able to make my review comments with a more relevant context. It may even be helpful to have PRs *always* use the "patience" diff algorithm.

**Comment by calsmith1-nokia** [comment]
Agreed
Seems similar to https://github.com/orgs/community/discussions/78241
